barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.hpp

```diff
@@ -519,7 +729,7 @@ template <typename Builder, typename T> class bigfield {
     static constexpr uint512_t get_maximum_unreduced_value()
     {
         // This = `T * n = 2^272 * |BN(Fr)|` So this equals n*2^t
-        uint1024_t maximum_product = uint1024_t(binary_basis.modulus) * uint1024_t(prime_basis.modulus);
+        uint1024_t maximum_product = get_maximum_crt_product();
```

Comment:
just reusing the `get_maximum_crt_product()` function instead of redundant definition